### PR TITLE
Surgery refactors

### PR DESCRIPTION
**Pull request by alex-gh**
## What Does This PR Do

Removes some dead code from surgery and refactors surgery by changing magic numbers to defines.

## Changelog

No player facing changes

### CODE REVIEW

File: code/modules/surgery/limb_augmentation.dm [hunk 1]
```diff
@@ -42,9 +42,9 @@
 		for(var/part_name in L.part)
 			if(!target.get_organ(part_name))
 				continue
-			affected.robotize(L.model_info, make_tough = 1, convert_all = 0)
+			affected.robotize(L.model_info, make_tough = TRUE, convert_all = FALSE)
```
Comment (moxian) [inline code/modules/surgery/limb_augmentation.dm:45]: May i ask to change `robotize` signature as well to have `TRUE`/`FALSE` rather than `1`/`0`?

**Review by moxian — APPROVED** [review]

File: code/modules/surgery/limb_augmentation.dm (hunk 1 above)
Comment (alex-gh) [reply code/modules/surgery/limb_augmentation.dm:45]: Missed that one, done!

**Review by moxian — APPROVED** [review]

File: code/modules/surgery/organs_internal.dm [hunk 2]
```diff
@@ -31,37 +31,36 @@
...
-		return 1
+			return FALSE
+		return TRUE
 
 /datum/surgery/organ_manipulation/alien/can_start(mob/user, mob/living/carbon/target)
 	if(istype(target,/mob/living/carbon/alien/humanoid))
-		return 1
-	else return 0
+		return TRUE
+	else return FALSE
```
Comment (farie82) [inline code/modules/surgery/organs_internal.dm:59]:
```suggestion
	return FALSE
```

**Review by farie82 — COMMENTED** [review]
Small nitpick. LGTM otherwise

File: code/modules/surgery/organs_internal.dm (hunk 2 above)
Comment (alex-gh) [reply code/modules/surgery/organs_internal.dm]: Fixed, thanks!

**Review by farie82 — APPROVED** [review]

**Pull request merged**